### PR TITLE
add example of `SteppedTracker` with a single label

### PR DESCRIPTION
**Pull request by mark-tate**

### CODE REVIEW

**Review by tomhazledine — DISMISSED** [review]
Ticks all the boxes from the design side (based on feedback from @ivan-calderon on #3485 )

Much cleaner with the `alignment` prop (vs. "compact" variant) 👍

File: packages/lab/src/stepped-tracker/SteppedTracker.tsx [hunk 1]
```diff
@@ -78,15 +83,20 @@ export const SteppedTracker = forwardRef<HTMLUListElement, SteppedTrackerProps>(
...
     return (
       <SteppedTrackerProvider totalSteps={totalSteps} activeStep={activeStep}>
-        <ul
-          className={clsx(withBaseName(), className, withBaseName(orientation))}
+        <ol
+          className={clsx(
+            withBaseName(),
+            className,
+            withBaseName(orientation),
+            withBaseName(`align${capitalize(alignment)}`)
```
Comment (joshwooding) [inline packages/lab/src/stepped-tracker/SteppedTracker.tsx:91]:
```suggestion
            withBaseName(),
            withBaseName(orientation),
            withBaseName(`align${capitalize(alignment)}`),
            className,
```

**Review by tomhazledine — APPROVED** [review]

File: site/src/examples/stepped-tracker/SingleLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        {steps.map(({ label, state }, key) => (
+          <TrackerStep
+            aria-labelledby="stepper_label"
+            state={state}
+            key={key}
+          ></TrackerStep>
+        ))}
+      </SteppedTracker>
+      <Text id="stepper_label" style={{ marginTop: "var(--salt-spacing-100)" }}>
+        Completed:
```
Comment (lilyvc) [inline site/src/examples/stepped-tracker/SingleLabel.tsx:79]: [Nit] there is no space in the example: "Completed:Step One" would look better as "Completed: Step One"

**Review by lilyvc — APPROVED** [review]

**Comment by mark-tate** [comment]
@tomhazledine  closing

**Pull request closed without merging**